attributes/default.rb

```diff
@@ -6,12 +6,13 @@
 default['amazon-ecs-agent']['tag'] = 'latest'
 default['amazon-ecs-agent']['user'] = 'ubuntu'
 default['amazon-ecs-agent']['storage_driver'] = 'overlay'
-default['amazon-ecs-agent']['docker']['version'] = '1.11.2'
+default['amazon-ecs-agent']['docker']['version'] = '17.05.0'
```

Comment:
We should definitely keep the Docker version updated but this is a big jump, we just need to make sure that we deploy and monitor this closely in QA first.